paasta_tools/check_chronos_jobs.py

```diff
@@ -34,6 +34,16 @@ def parse_args():
     return args
 
 
+def _guess_realert_every(chronos_job_config):
+    """Returns the job interval (in minutes) or None if the job has no interval.
+
+    As we run check_chronos_jobs once a minute, we are going to skip
+    all re-alert attempts until the next scheduled run of the job.
+    """
+    interval = chronos_job_config.get_schedule_interval_in_seconds()
+    return (interval // 60) if interval is not None else None
```

Comment:
'//' is intentional, because the result of (interval // 60) is int in both python2.7 and python3.
While (interval / 60) is float in python3.

Just to be sure, do you want to clamp it as max(i, 24 * 60 * 60) or min(i, 24 * 60 * 60)?